### ISSUE TITLE
Option to donate - what do you think?

### ISSUE THREAD

**Issue by jendabek** [Isti115/BlenderRenderController#8]
Hello, as I plan to make my fork more public, I think it would be shame to not use an option to donate to support the further development.
Even I have almost rewritten it completely, I realize the main idea / core of the app functionality was not born in my head.
So I would like to ask you what solution do you suggest?
Can I point it to my PayPal and give other contributors some portion of the donated amount?
Thanks

**Comment by red-rj** [comment]
Like I said before, I'm fine with donations as long as the division is fair and everyone else agrees.

We should also think of getting the world about BRC out there, no point having a donate button w/o an audience. @jendabek's fork is shaping up really nice, it has a lot of sweet features. People should know about it 😄

**Comment by jendabek** [comment]
So I suggest to add that donate button to the app while I will send corresponding division of received donations to you. Please suggest an appropriate ratio.
I am going to get know about the tool to public via simple video on YT and some post on Blender Artists forum. It could also be worth to make a note about it in some relevant topics on Blender Stack Exchange.

**Comment by red-rj** [comment]
I've sent an E-mail to @Isti115 to let him know about it. I want to hear what he has to say about it.

I mean, I wrote the readme, did a lot of work on the old version before you showed up, even did some commits on your fork. 

I would suggest a simple split. And everybody is happy

**Comment by isti115** [comment]
Sorry for such a late answer, but I've been really busy this week.

So, about the option to donate: I completely stand by the idea, as I can see how it would motivate you to continue working on this, and thus further improve the program. :)

About the split: As you have already said, compared to your work my amount of initial code is really small, so the only significant thing is the actual idea itself. I wouldn't like to ask for much, let's say about 10%. That's practically nothing lost for you and at least a symbolic amount for me.
As for distributing the remaining 90% between the two of you my only source of information was the contributors page on GitHub. By looking at that data I would suggest the following:

@jendabek : 60%
@RedRaptor93 : 30%
(@isti115 : 10%)

If any of you feel like this isn't fair, please write here and we can discuss it. Again, this is based on the current state of the software, so if in the future @RedRaptor93, you decide to contribute more, you could always adjust the ratio accordingly.

The only extra thing I would like to ask for would be to link the original blender stackexchange answer into the readme as well as the youtube description, so that they can learn about the process of discovery there. 😉 

ps.: @jendabek, I have watched the tutorial video, and it is very nice! :) I hope that it will help lots of people get started using *BRC*! Just one suggestion: I would test the performance difference caused by reading the frame counts being rendered. Something in the back of my mind is telling me that it could be slowing down the process, but I might be wrong.

**Comment by jendabek** [comment]
Yes, I have read something about that possible performance issue when reading the output. I think the advantage of being able to see the progress accuratelly is more important than possible impact to the performance though. But I will have to test it to see how much difference it makes in reality.
I can imagine that I could read the output in longer intervals so it won't catch every frame - I could just fake/interpolate the progress between them.
There is also an issue that not all the frames are actually catched if the rendering is too fast so I definitely need to look at the progress feature anyway.

**Comment by red-rj** [comment]
I'm fine w/ the @Isti115's proposed split.

Unfortunately I too will be busy starting tomorrow as I go back to collage... But I'll will participate in developing when ever possible.
